modules/custom/localgov_restricted_content/templates/localgov-restricted-content-sign-in-with-message.html.twig

```diff
@@ -0,0 +1,8 @@
+<!--cludooff: index-->
```

Comment:
`cludooff/cludoon` would be Essex specific. So if it becomes an LGD module, I suppose there would need to be a theme override?